Indexing/CodeTree.cpp

```diff
@@ -1325,7 +1325,6 @@ bool CodeTree::RemovingMatcher::next()
       if(!backtrack()) {
 	return false;
       }
-      shouldBacktrack=false;
```

Comment:
What about this - is it just never read?

Reply:
Yeah, dead-store elimination caught it.

Reply:
Ah - unless this is another should-have-been-a-static case @selig ?

Reply:
Bleh, it's been many years since I've looked at CodeTree. Can we make it a comment-out instead of a delete in case it helps somebody in the future to know that it was there in the past!

Reply:
Done!